CHANGES.txt

```diff
@@ -22,6 +22,9 @@ Changes
 Fixes
 =====
 
+ - Fixed `IS NULL` and `IS NOT NULL` directly on an array of objects by using
```

Comment:
I think this sentence doesn't make much sense to usesr who don't know about the internals